package.json

```diff
@@ -22,7 +22,7 @@
     "@changesets/cli": "^2.26.1",
     "prettier": "^3.2.5",
     "turbo": "1.10.2",
-    "typescript": "^5.2.2",
+    "typescript": "^5.5.4",
```

Comment:
Unrelated to the PR, but I was getting a wonky TS error somewhere and updating TS fixed it.